script.js

```diff
@@ -0,0 +1,245 @@
+// eng key lang
+
+const keyboardEng = [
```

Comment:
Well, this is just strange, I expect `keyboardEng` and `keyboardRus` arrays to be one shape, but they are different. For sure can be refactored